### PR TITLE
docs: Add drupal9/11 tabs and symfony to quickstart

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

* Add drupal9/11 tabs to quickstarts
* Add symfony

Rendered versions at:
* https://ddev--6060.org.readthedocs.build/en/6060/users/quickstart/#drupal
* https://ddev--6060.org.readthedocs.build/en/6060/users/quickstart/#symfony

### CODE REVIEW

File: docs/content/users/quickstart.md [hunk 1]
```diff
@@ -162,6 +162,7 @@ ddev launch
 ## Drupal
 
 === "Drupal"
```
Comment (rpkoller) [inline docs/content/users/quickstart.md:164]: It reads odd to have at first a tab called drupal and then tabs with specific version numbering and technically the order is 10, 11, 9, 6/7... plus it feels odd to have the explanation inside the box on just one out of the three tabs this explanation applies to. I wonder if the following would make sense. 

move ```For all versions of Drupal 8+ the Composer techniques work. The settings configuration is done differently for each Drupal version, but the project type is “drupal”.``` outside the box, between the title Drupal and the box with the drupal tabs. and in regards of the tab naming and ordering, how about switching the position between drupal 10 and 11 and renaming `drupal` to `Drupal 10` and `Drupal 11` to `Drupal 11 (dev)` (or something similar)?

File: docs/content/users/quickstart.md [hunk 2]
```diff
@@ -587,6 +614,30 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
...
+=== "Git Clone"
+
+    ```bash
+    git clone <your-symfony-repo>
+    cd <your-symfony-repo>
+    ddev config --docroot=public --php-version=8.3
+    ddev start
+    ddev composer install
+    ddev launch
+    ```
```
Comment (penyaskito) [inline docs/content/users/quickstart.md:639]: I'd suggest a couple of links with more information until we can have a symfony project-type 

```suggestion
    ddev launch
    ```
As Symfony projects can be so flexible, you might need different customizations for your project.

You might want to start by editing your `.env` file for setting the DB connection string if you require one. See the [official docs](https://symfony.com/doc/current/doctrine.html#configuring-the-database) for more information.
If you require your current database engine version, you can use `ddev -s db exec --raw mysql -udb -pdb db -e 'SELECT VERSION();'`

Read more about customizing the environment with different [Config Options](https://ddev.readthedocs.io/en/latest/users/configuration/config/) and [Defining Additional Services with Docker Compose](https://ddev.readthedocs.io/en/latest/users/extend/custom-compose-files/).    
    
```

**Review by rpkoller — COMMENTED** [review]
and the quickstarts for drupal we should revisit in a follow up issue  when the `ddev config --update`PR is finalized imho

File: docs/content/users/quickstart.md (hunk 1 above)
Comment (rfay) [reply docs/content/users/quickstart.md:164]: It's hard to follow your request. Maybe you can do a mockup, or just edit as you see fit and show what happens with `make mkdocs-serve`

File: docs/content/users/quickstart.md (hunk 2 above)
Comment (rfay) [reply docs/content/users/quickstart.md:639]: I guess you don't know about `ddev mysql`. You'll like it! Please edit to use it. 

But... I don't understand what "if you require your current database engine version" means. Wouldn't they just `ddev describe` ?

The links should be relative links (in docs) instead of explicit links. Can you edit?

Comment (penyaskito) [reply docs/content/users/quickstart.md:639]: Updated those.

The version suggestion is because by default doctrine uses a DSN, and looks like it's a bit picky about those with mariadb/mysql according to the official Doctrine DBAL docs, so you might want to include the version in the DSN connection string.

https://www.doctrine-project.org/projects/doctrine-dbal/en/4.0/reference/configuration.html#automatic-platform-version-detection

File: docs/content/users/quickstart.md (hunk 1 above)
Comment (rpkoller) [reply docs/content/users/quickstart.md:164]: Screenshot 2024-04-07 at 03 01 57

I meant to move the description out of the drupal tab and perhaps slightly adjust it to something like: 

```
For all versions of Drupal 8+ the installation is done via Composer. The settings configuration is different for every version of Drupal, but the project type remains just "drupal".
```
and changing the tab labels  to something like:  

Drupal 11 (dev), Drupal 10, Drupal 9, Drupal 6/7,  Git Clone

File: docs/content/users/quickstart.md (hunk 2 above)
Comment (rfay) [reply docs/content/users/quickstart.md:639]: Just `ddev mysql  -e 'SELECT VERSION();'` :)

But not sure how it's better than `ddev describe`

Comment (penyaskito) [reply docs/content/users/quickstart.md:639]: It gives the exact major.minor.patch version. Honestly, don't know how relevant that is, but most of examples in those docs specify at that level.

File: docs/content/users/quickstart.md [hunk 3]
```diff
@@ -175,16 +176,36 @@ ddev launch
...
+    ddev start
+    ddev composer create drupal/recommended-project:^9
+    ddev composer require drush/drush
+    ddev drush site:install --account-name=admin --account-pass=admin -y
+    # use the one-time link (CTRL/CMD + Click) from the command below to edit your admin account details.
+    ddev drush uli
+    ddev launch
     ```
 
 === "Drupal 6/7"
```
Comment (rpkoller) [inline docs/content/users/quickstart.md:211]:
```suggestion
=== "Drupal 6 (EOL)/7"
```
A real good idea adding EOL to Drupal 9! would it make sense to clarify that for drupal 6 as well?

File: docs/content/users/quickstart.md (hunk 2 above)
Comment (rfay) [reply docs/content/users/quickstart.md:666]: I added a commit with your suggestion and minor changes.

File: docs/content/users/quickstart.md (hunk 1 above)
Comment (rfay) [reply docs/content/users/quickstart.md]: It's now like this: 
image

File: docs/content/users/quickstart.md (hunk 3 above)
Comment (rfay) [reply docs/content/users/quickstart.md:211]: IMO they're so long gone that people should know this themselves, and will be using them deliberately if needed.

File: docs/content/users/quickstart.md [hunk 4]
```diff
@@ -449,23 +470,27 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
...
-!!!tip
-    Moodle relies on a periodic cron job—don’t forget to set that up! See [ddev/ddev-cron](https://github.com/ddev/ddev-cron).
+    Visit the [Moodle Admin Quick Guide](https://docs.moodle.org/400/en/Admin_quick_guide) for more information.
+
+    !!!tip
+        Moodle relies on a periodic cron job—don’t forget to set that up! See [ddev/ddev-cron](https://github.com/ddev/ddev-cron).
 
 ## Pimcore
 
+=== "Composer"
```
Comment (stasadev) [inline docs/content/users/quickstart.md:492]: Formatting is broken, please add tabs for the text below.

File: docs/content/users/quickstart.md [hunk 5]
```diff
@@ -587,6 +614,57 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
...
+    ```bash
+    RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | sudo -E bash
+    RUN sudo apt install -y symfony-cli
+    ```
+
+    Then:
+
+    ```bash
+    mkdir my-symfony && cd my-symfony
+    ddev config --docroot=public
```
Comment (stasadev) [inline docs/content/users/quickstart.md:649]: We can use `echo` to add the file, as in other quickstarts:

````suggestion
    ```bash
    mkdir my-symfony && cd my-symfony
    ddev config --docroot=public
    echo "RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | sudo -E bash
    RUN sudo apt install -y symfony-cli" >.ddev/web-build/Dockerfile.symfony-cli
````

**Review by stasadev — CHANGES_REQUESTED** [review]
Tested Symfony quickstart. Works great 👍

I only have suggestions for formatting.

**Comment by sanjaybip** [comment]
Looks good to me.

File: docs/content/users/quickstart.md (hunk 5 above)
Comment (sanjaybip) [reply docs/content/users/quickstart.md:649]: Why this? Is  this better?

Comment (stasadev) [reply docs/content/users/quickstart.md:649]: Because the order of actions is wrong:

The example starts by creating `.ddev/web-build/Dockerfile.symfony-cli` and then you create a project folder
```
mkdir my-symfony && cd my-symfony
ddev config --docroot=public
```

Comment (sanjaybip) [reply docs/content/users/quickstart.md:649]: Make sense.

**Comment by rfay** [comment]
I think I got everything @stasadev , thanks!

**Comment by rfay** [comment]
Thanks all, I pulled so I could build on these docs in
* #6055
